js_modules/dagit/packages/core/src/hooks/useQueryPersistedState.tsx

```diff
@@ -96,7 +97,6 @@ export function useQueryPersistedState<T extends QueryPersistedDataType>(
         }
       }
 
-      currentQueryString = next;
```

Comment:
It was never necessary since this is the last reference to `currentQueryString` in this closure and no child closures are referencing it, so it was never doing anything or being used by anything.

Reply:
oh wait, it's being set up by the parent closure. Yeah I can add this back. Mistakenly thought it wasn't necessary